build.gradle

```
@@ -17,6 +17,7 @@ dependencies {
     implementation 'org.springframework.boot:spring-boot-starter-web'
     implementation 'org.springframework:spring-orm'
     implementation "org.hibernate.orm:hibernate-core:6.3.0.Final"
```

https://github.com/spring-projects/spring-framework/issues/31155#issuecomment-1751699188
```suggestion
    implementation "org.hibernate.orm:hibernate-core:6.3.1.Final"
```

В даному випадку лише гугл допоміг. Треба оновити бібліотеку.